databricks_cli/dbfs/api.py

```diff
@@ -97,20 +119,37 @@ def file_exists(self, dbfs_path, headers=None):
             raise e
         return True
 
+    @retry_429
     def get_status(self, dbfs_path, headers=None):
         json = self.client.get_status(dbfs_path.absolute_path, headers=headers)
         return FileInfo.from_json(json)
 
+    @retry_429
+    def create(self, dbfs_path, overwrite, headers):
+        return self.client.create(dbfs_path, overwrite, headers=headers)
```

Comment:
Here we need to pass `dbfs_path.absolute_path` to `self.client.create`, right?

Reply:
Hmm, weird how the tests were passing with this.

Reply:
II think they were passing because `self.client.create` was mocked and did not get to execute code that would break.